Description: Allow pyinotify 0.9.5
 Debian still has 0.9.5, and the latest upstream release doesn't contain
 so critical fixes, so we allow a lower version.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2016-01-15

--- a/requirements.txt
+++ b/requirements.txt
@@ -10,5 +10,5 @@
 oslo.utils>=3.11.0 # Apache-2.0
 oslo.serialization>=1.10.0 # Apache-2.0
 debtcollector>=1.2.0 # Apache-2.0
-pyinotify>=0.9.6;sys_platform!='win32' and sys_platform!='darwin' and sys_platform!='sunos5' # MIT
+pyinotify>=0.9.5;sys_platform!='win32' and sys_platform!='darwin' and sys_platform!='sunos5' # MIT
 python-dateutil>=2.4.2 # BSD
